ee/codegen/src/generators/nodes/code-execution-node.ts

```diff
@@ -95,7 +101,10 @@ export class CodeExecutionNode extends BaseSingleFileNode<
         python.field({
           name: CODE_INPUT_KEY,
           initializer: python.TypeInstantiation.str(this.scriptFileContents, {
-            multiline: true,
+            // Fern will garble the escaping when using python multiline for inline so we disable it.
+            // Inline code generation is currently only used by vembda and likely to remain that way
+            // so skipping over pretty formatting is acceptable.
+            multiline: false,
```

Comment:
[blocking] could we make our disabling of multiline here more targeted? eg if we detect `"\\"` in the `scriptFileContents`

same for the skipping of formatting above

Reply:
I don't think this is really blocking because this case is really only for internal vembda usage. Users are very unlikely to want inline code since it makes it unreadable with all the escapes.

Reply:
Also this way might catch other cases we might not have thought about

Reply:
ok I'll downgrade to a [nit] - in general I want to strive for the gap between vembda representation and local representations as minimal as possible, and one day be zero

Reply:
I would say in this case we would need to be able to use the script files directly to match 1:1 I forget where we landed on that to make it happen? Don't think that would be to bad actually just write a temp file for it to load and a special if check somewhere to know where to read it from

Reply:
Yea I think we're currently blocked on the fact that we are using `os` and some bugs with `inspect`. We are tracking solving both here https://linear.app/vellum/issue/APO-239/remove-disparity-between-local-code-node-resolution-and-vembda-code, which we could consider [oos] of this PR